sdk/translation/ai-translation-text/review/azure-ai-translation-text.api.md

```diff
@@ -0,0 +1,643 @@
+## API Report File for "@azure-rest/ai-translation-text"
```

Comment:
Looks like a duplicate file, could you please delete it?

Reply:
Deleted